subsys/bluetooth/host/keys.h

```diff
@@ -102,3 +105,7 @@ struct bt_keys_link_key *bt_keys_get_link_key(const bt_addr_t *addr);
 struct bt_keys_link_key *bt_keys_find_link_key(const bt_addr_t *addr);
 void bt_keys_link_key_clear(struct bt_keys_link_key *link_key);
 void bt_keys_link_key_clear_addr(const bt_addr_t *addr);
+
+#if (defined(CONFIG_BT_KEYS_OVERWRITE_OLDEST))
+void bt_keys_notify_usage(u8_t id, const bt_addr_le_t *addr);
+#endif /* CONFIG_BT_KEYS_OVERWRITE_OLDEST */
```

Comment:
No need to have `#if ... #endif` around function declarations in header files. Actually, it's not just that there's no need, you're also actively preventing the use of the `IS_ENABLED()` macro if you don't allow the c-files to see the prototype always.